atomate/vasp/drones.py

```diff
@@ -103,7 +104,7 @@ def __init__(self, runs=None, parse_dos="auto", compress_dos=False, bandstructur
         self.compress_dos = compress_dos
         self.additional_fields = additional_fields or {}
         self.use_full_uri = use_full_uri
-        self.runs = runs or ["relax" + str(i + 1) for i in range(9)]  # can't auto-detect: path unknown
+        self.runs = runs or ["relax" + str(i) for i in range(9)]  # can't auto-detect: path unknown
```

Comment:
do we have relax0?

I thought we had relax1, relax2, etc.

Reply:
The scan_opt_job that I put into custodian has a relax0 task. This is just a pre-calculation of the Wavefunctions using regular GGA that is used to condition the electronic SCF cycle when running a metaGGA calculation. This was just the simplest solution between the two and @shyuep  didn't complain about it there. I can change both so that this step is named something else:  "e_condition" ?